config.js

```diff
@@ -187,4 +187,7 @@ try {
     const _ = require('lodash');
     _.merge(module.exports, projectConfig);
   }
-} catch(e) {}
+} catch(e) {
+    const colors = require('colors/safe');
+    console.log(colors.red('Error: ', e));
```

Comment:
I would like to change this to a more descriptive error message. 
What about 
```
console.log(colors.red(`Error in local project config "${projectConfig}": ${e.message}`));
```

Reply:
I prefer logging e instead of e.message, because e.message does not tell us where exactly the error is coming from while e does. Also note that if you wish to display projectConfig's value in log, then you have to refactor try/catch snippet so projectConfig is not only in try's scope.

Reply:
You're right. OK for me